### DISCUSSION TITLE
Where do libvips and net-vips look for fonts?

### DISCUSSION THREAD

**Discussion by stanley-zenfolio** [libvips/libvips#2210] [General]
Where do libvips and net-vips look for fonts?
If font specified in Image.Text(...) isn't found, does a default font used?

**Comment by jcupitt** [comment]
Hi @stanley-zenfolio,

libvips uses fontconfig:

https://www.freedesktop.org/wiki/Software/fontconfig/

So it depends on your platform and how it's been configured. On linux, for example, it usually searches the tree under `/usr/share/fonts`, and your personal `~/.fonts` area.

Yes, the name you give to fontconfig is a general specification, so for example `"sans 12"` will pick the "best" sans-serif font on your system.
